.pre-commit-config.yaml

```diff
@@ -20,7 +20,6 @@ repos:
     rev: 'v0.0.271'
     hooks:
       - id: ruff
-        files: ^(great_expectations|contrib|scripts|tasks\.py) # TODO: add tests/  docs/ etc.
```

Comment:
This change will cause the ruff hook to be run on all python files.
This might cause some friction if users have pre-commit installed and they are modifying one of the few remaining folders that have pre-existing linting errors.